b2sdk/file_version.py

```diff
@@ -86,7 +86,13 @@ def __init__(
         self.replication_status = replication_status
 
         if SRC_LAST_MODIFIED_MILLIS in self.file_info:
-            self.mod_time_millis = int(self.file_info[SRC_LAST_MODIFIED_MILLIS])
+            try:
+                if SRC_LAST_MODIFIED_MILLIS in self.file_info:
+                    self.mod_time_millis = int(self.file_info[SRC_LAST_MODIFIED_MILLIS])
```

Comment:
```suggestion
                self.mod_time_millis = int(self.file_info[SRC_LAST_MODIFIED_MILLIS])
```
the `if` statement was repeated from line 88